scheduler/views.py

```diff
@@ -43,10 +43,20 @@ def courseManagement(request):
                 return render(request, 'courseManagement.html', {'courses': courses, 'TAs': TAs, 'instructors': instructors, 'labs': labs, 'messages': "Course successfully created"})
             except ValueError as msg:
                 return render(request, 'courseManagement.html', {'courses': courses, 'TAs': TAs, 'instructors': instructors, 'labs': labs, 'messages': msg})
+        if request.method == 'POST' and 'editCourseBtn' in request.POST:
```

Comment:
It doesn't look like you have the latest views, and I think this is going to be a conflict. Can you pull and merge the latest code review into your branch? So then there wont be problems

Reply:
oh youre merging to master not code review, thats why its not showing there's conflicts

Reply:
i changed the pull request to code-review instead of master